Frends.Excel.ConvertToCSV/Frends.Excel.ConvertToCSV/Input.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.ComponentModel;
```

Comment:
Why are task parameters in their own classes and not in Definitions.cs?

Reply:
I never honstly understood why we should cram everything into Definitions.cs, seems very weird way of conserving the number of files in the repository. I am pretty sure that most best practices do still say "1 class per file", which I completely agree with.

However, I think that it would be nice to have a folder for the definitions that would contain all those, so that the root folder in project is clean and does not contain too much files.